src/coreclr/vm/method.hpp

```diff
@@ -1437,9 +1437,9 @@ class MethodDesc
     PCODE GetNativeCode();
 
     // Returns GetNativeCode() if it exists, but also checks to see if there
-    // is a non-default IL code version and returns that.
+    // is a non-default code version that is populated with a code body and returns that.
     // Perf warning: takes the CodeVersionManagerLock on every call
-    PCODE GetNativeCodeReJITAware();
```

Comment:
There's no callers of `GetNativeCodeAnyVersion` other than `HasNativeCodeAnyVersion` which checks it's not null. Any reason not to fold it in?

Reply:
If we are talking about refactoring in the development branch, I think it would be fine, its a stylistic choice. If we are talking about what gets committed for release/8.0 I think we should leave it as-is to minimize churn.